gatsby-node.js

```diff
@@ -5,42 +5,73 @@
  */
 const path = require('path');
 
-const ProgramPage = path.resolve('./src/components/masters/program/ProgramPage.jsx');
+const programPageType = 'pages.ProgramPage';
+const programListingTemplate = path.resolve('../src/components/masters/programs-list/ProgramListPage.jsx');
+const programPageTemplate = path.resolve('./src/components/masters/program/ProgramPage.jsx');
 
-exports.createPages = async ({ actions }) => {
+
+const tranformPageContext = context => (
```

Comment:
nit: should we name this to be clear its for the `ProgramPage`? Maybe `transformProgramPageContext`?